packages/p2p/src/components/page-return/page-return.tsx

```diff
@@ -8,7 +8,7 @@ type TPageReturnProps = {
     page_title: string;
 };
 
-const PageReturn = ({ className = '', onClick, page_title }: TPageReturnProps) => {
+const PageReturn: React.FC<TPageReturnProps> = ({ className = '', onClick, page_title }) => {
```

Comment:
`nit` i think its better just to leave it as it was before since we are doing it already in other components, so just to have consistency

```suggestion
const PageReturn = ({ className = '', onClick, page_title }: TPageReturnProps) => {
```

Reply:
but won't we be adding the React.FC<> for the new ones that we will be refactoring? i just saw in one of the read me files in our deriv app to use React.FC<>